src/CoreBL/Models/Car.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+
+namespace CoreBL.Models
+{
+    public class Car
+    {
+        private DateTime _releaseDate;
+        
+        public Guid Id { get; set; }
+
+        [Required]
+        [MinLength(5)]
+        [MaxLength(12)]
+        public string RegistrationPlate { get; set; }
+
+        [Required]
+        [MinLength(2)]
+        [MaxLength(25)]
+        public string Brand { get; set; }
+
+        [Required]
+        public DateTime ReleaseDate
+        {
+            get => _releaseDate;
+            set
+            {
+                if (!DateTime.TryParse(value.ToString(CultureInfo.InvariantCulture), out _releaseDate))
```

Comment:
Did you check does TryParse rewrite `_releaseDate` even in case of failing parse?

Reply:
how to check it?
we have 2 preventions:
1) before entering the controller, the entire model is checked
2) in `TryParse` resultData = parsedDate or DateTime.MinValue, but I don't know how to create a situation to get `result = DateTime.MinValue`